frontend/lib/DataModels/field_data.dart

```diff
@@ -0,0 +1,43 @@
+//ignore_for_file:  constant_identifier_names
+enum FieldType { FOR_CHART, STATIC_NUMBER, STRING, ERROR }
```

Comment:
niemal to samo mamy w pliku enable_field.dart, zerknij czy nie da się tego jakoś połączyć, a jak nie to zostaw